ipn/ipnserver/server.go

```diff
@@ -187,13 +197,25 @@ func Run(ctx context.Context, logf logger.Logf, logid string, opts Options, e wg
 				bo.BackOff(ctx, err)
 				continue
 			}
-			serverToClient := func(b []byte) {
-				ipn.WriteMsg(s, b)
+			errMsg := opts.ErrorMessage
+			if opts.GetEngine != nil {
+				logf("%d: trying GetEngine again...", i)
+				eng, err = opts.GetEngine()
```

Comment:
staticcheck mad, please fix?

Reply:
Filing a bug with staticcheck, adding ignore directive for now.